Podfile

```diff
@@ -23,3 +25,34 @@ abstract_target 'Automattic' do
   #
   target 'SimplenoteTests'
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    # Let Pods targets inherit deployment target from the app
+    # See https://github.com/CocoaPods/CocoaPods/issues/4859
+    target.build_configurations.each do |configuration|
+      macos_deployment_key = 'MACOSX_DEPLOYMENT_TARGET'
+      pod_macos_deployment_target = Gem::Version.new(configuration.build_settings[macos_deployment_key])
+      if pod_macos_deployment_target <= APP_MACOS_DEPLOYMENT_TARGET
+        configuration.build_settings.delete macos_deployment_key
+      end
+    end
+
+    # Workaround for Xcode 14.3 archiving issue with CocoaPods.
+    # See https://github.com/CocoaPods/CocoaPods/issues/11808#issuecomment-1500907375
+    #
+    # What this does is updating the generated frameworks to call `readlink`
+    # with the `-f` option, which is apparently necessary to match the file
+    # structure generated by Xcode 14.3
+    shell_script_path = "Pods/Target Support Files/#{target.name}/#{target.name}-frameworks.sh"
+    next unless File.exist?(shell_script_path)
+
+    shell_script_input_lines = File.readlines(shell_script_path)
+    shell_script_output_lines = shell_script_input_lines.map { |line| line.sub("source=\"$(readlink \"${source}\")\"", "source=\"$(readlink -f \"${source}\")\"") }
+    File.open(shell_script_path, 'w') do |f|
+      shell_script_output_lines.each do |line|
+        f.write line
+      end
+    end
+  end
+end
```

Comment:
I just noticed CocoaPods [released 1.12.1](https://github.com/CocoaPods/CocoaPods/releases/tag/1.12.1) 4 hours ago, have you tried the latest version which hopefully fixed these integration issues in Xcode 14.3?

Reply:
Neat! I'll try it out, it does mention the fix we need:

<img width="754" alt="Screenshot 2023-04-19 at 3 55 56 pm" src="https://user-images.githubusercontent.com/1218433/232979777-ef69f7e6-1e9d-4fec-aae0-9742880c1173.png">